src/webgpu/api/validation/capability_checks/limits/maxSampledTexturesPerShaderStage.spec.ts

```diff
@@ -0,0 +1,133 @@
+      limitTest,
+      testValueName,
+      async ({ device, testValue, shouldError }) => {
+        await t.expectValidationError(() => {
+          device.createBindGroupLayout({
+            entries: reorder(
+              order,
+              range(testValue, i => ({
+                binding: i,
+                visibility,
+                texture: {},
+              }))
+            ),
+          });
+        }, shouldError);
+      }
+    );
+  });
+
+g.test('createPipeline,at_over')
+  .desc(
+    `
+  Test using createRenderPipeline and createComputePipeline at and over ${limit} limit
+  
+  Note: We also test order to make sure the implementation isn't just looking
+  at just the last entry.
+  `
+  )
+  .params(
+    kLimitBaseParams
+      .combine('bindingCombination', kBindingCombinations)
+      .combine('order', kOrderKeys)
+      .combine('bindGroupTest', kBindGroupTests)
+  )
+  .fn(async t => {
+    const { limitTest, testValueName, bindingCombination, order, bindGroupTest } = t.params;
+    const pipelineType = getPipelineTypeForBindingCombination(bindingCombination);
+
+    await t.testDeviceWithRequestedLimits(
+      limitTest,
+      testValueName,
+      async ({ device, testValue, shouldError }) => {
+        const code = getPerStageWGSLForBindingCombination(
+          bindingCombination,
+          order,
+          bindGroupTest,
+          (i, j) => `var u${j}_${i}: texture_2d<f32>`,
+          (i, j) => `textureLoad(u${j}_${i}, vec2u(0), 0).x`,
+          testValue
+        );
+        const module = device.createShaderModule({ code });
+
+        await t.expectValidationError(() => {
+          t.createPipeline(pipelineType, module);
+        }, shouldError);
+      }
+    );
+  });
+
+g.test('createPipelineAsync,at_over')
+  .desc(
+    `
+  Test using createRenderPipelineAsync and createComputePipelineAsync at and over ${limit} limit
+  
+  Note: We also test order to make sure the implementation isn't just looking
+  at just the last entry.
+  `
+  )
+  .params(
+    kLimitBaseParams
+      .combine('bindingCombination', kBindingCombinations)
+      .combine('order', kOrderKeys)
+      .combine('bindGroupTest', kBindGroupTests)
+  )
+  .fn(async t => {
+    const { limitTest, testValueName, bindingCombination, order, bindGroupTest } = t.params;
+    const pipelineType = getPipelineAsyncTypeForBindingCombination(bindingCombination);
+
+    await t.testDeviceWithRequestedLimits(
+      limitTest,
+      testValueName,
+      async ({ device, testValue, shouldError }) => {
+        const code = getPerStageWGSLForBindingCombination(
+          bindingCombination,
+          order,
+          bindGroupTest,
+          (i, j) => `var u${j}_${i}: texture_2d<f32>`,
+          (i, j) => `textureLoad(u${j}_${i}, vec2u(0), 0).x`,
+          testValue
+        );
+        const module = device.createShaderModule({ code });
+
+        await t.shouldRejectConditionally(
+          'OperationError',
```

Comment:
ditto GPUPipelineError here and in other files.

Reply:
When I change it I get

> THREW OperationError, instead of GPUPipelineError:

Is that bug or should the test stay `OperationError`?

Reply:
It is a Chromium bug fixed by https://chromium-review.googlesource.com/c/chromium/src/+/4277962